base/compiler/ssair/ir.jl

```diff
@@ -1390,7 +1410,7 @@ function process_node!(compact::IncrementalCompact, result_idx::Int, inst::Instr
         if cfg_transforms_enabled
             # Rename phi node edges
             let bb_rename_pred=bb_rename_pred
-                map!(i::Int32->bb_rename_pred[i], stmt.edges, stmt.edges)
+                map!(i::Int32->i == 0 ? 0 : bb_rename_pred[i], stmt.edges, stmt.edges)
```

Comment:
Is this change necessary for this PR, or a fix for some related issues?

Reply:
It fixed some test failure, but I don't think it's related to this PR other than by coincidence. There's a special `i == 0` predecessor for entry into catch blocks.